Potions.rb

```diff
@@ -4,6 +4,9 @@ def initialize
   end
   def give_effect(target)
     target.HealDmg(10)
+    puts "You feel reinvigorated and ready for more combat!"
+    sleep(1)
```

Comment:
For game dev puposes we should comment out these sleep commands, they get really frustrating when testing...